v2/api/apimanagement/v1api20220801/structure.txt

```diff
@@ -0,0 +1,573 @@
+// Code generated by azure-service-operator-codegen. DO NOT EDIT.
+github.com/Azure/azure-service-operator/v2/api/apimanagement/v1api20220801
+├── APIVersion: Enum (1 value)
+│   └── "2022-08-01"
+├── Service: Resource
+│   ├── Owner: github.com/Azure/azure-service-operator/v2/api/resources/v1apiv20191001.ResourceGroup
+│   ├── Spec: Object (24 properties)
+│   │   ├── AdditionalLocations: Object (7 properties)[]
+│   │   │   ├── DisableGateway: *bool
+│   │   │   ├── Location: *string
+│   │   │   ├── NatGatewayState: *Enum (2 values)
+│   │   │   │   ├── "Disabled"
+│   │   │   │   └── "Enabled"
+│   │   │   ├── PublicIpAddressId: *string
+│   │   │   ├── Sku: *Object (2 properties)
+│   │   │   │   ├── Capacity: *int
+│   │   │   │   └── Name: *Enum (6 values)
+│   │   │   │       ├── "Basic"
+│   │   │   │       ├── "Consumption"
+│   │   │   │       ├── "Developer"
+│   │   │   │       ├── "Isolated"
+│   │   │   │       ├── "Premium"
+│   │   │   │       └── "Standard"
+│   │   │   ├── VirtualNetworkConfiguration: *Object (1 property)
+│   │   │   │   └── SubnetResourceId: Validated<*string> (1 rule)
```

Comment:
The validation rule on this has me thinking this property should be flagged with `$armReference: true` in `azure-arm.yaml`.

Reply:
Great - added this!